src/aliases/api/types/alias.go

```diff
@@ -0,0 +1,61 @@
+package types
+
+import aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+
+type Alias struct {
```

Comment:
as far as I see `Alias` is defined for reusability, so what you expose here are the Request and Response structs. Therefore I would suggest to make this stuct not exposed. In addition when you define you Request and Response would should define them with a pointer so you don't end up with nested attributes instead of flat object

```
type CreateAliasRequest struct {
	*Alias
}
```

Reply:
Can we have a call about this, I'm not sure I'm getting everything from this :)